css/components/input.css

```diff
@@ -0,0 +1,12 @@
+.input-field {
+  padding: 0.6rem;
+  border: 2px solid var(--secondary--color);
+  border-radius: 10px;
+  width: min-content;
+}
+.highliter {
+  border: 2px solid var(--border-danger);
+}
+.text-highliter {
```

Comment:
add 1 line space after each css block. it looks cleaner to read, just a suggestion

Reply:
Sure will implement this for all CSS files after completing all components